packages/account-abstraction/src/HttpRpcClient.ts

```diff
@@ -43,7 +43,7 @@ export class HttpRpcClient {
     // await this.initializing
     const hexifiedUserOp = deepHexlify(await resolveProperties(userOp1))
 
-    const jsonRequestData: [UserOperation, string] = [hexifiedUserOp, this.entryPointAddress]
+    // const jsonRequestData: [UserOperation, string] = [hexifiedUserOp, this.entryPointAddress]
```

Comment:
it's not used anywhere? also in the reference aa sdk?

Reply:
its used in aa sdk but the purpose is to just print request data

Reply:
ok